### PR TITLE
:arrow_up::one::one:, pos_invoice_postponed

### PR DESCRIPTION
**Pull request by fedoranvar**
In models.py added variable 'invoice_flag';
Deleted unused variables;
Edited commentaries in files (i.e. changed version from 10 to 11, added contributors etc.);

### CODE REVIEW

**Review by KolushovAlexandr — APPROVED** [review]

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
/static/description/index.html contains a reference to Odoo 10.0

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by kaadevelop — COMMENTED** [review]
No new features need to be tested.

File: pos_invoice_postponed/README.rst [hunk 1]
```diff
@@ -36,14 +37,14 @@ Maintainers
...
 
 Usage instructions: `<doc/index.rst>`_
 
 Changelog: `<doc/changelog.rst>`_
 
-Notifications on updates: `via Atom <https://github.com/it-projects-llc/pos-addons/commits/10.0/.atom>`_, `by Email <https://blogtrottr.com/?subscribe=https://github.com/it-projects-llc/pos-addons/commits/10.0/.atom>`_
+Notifications on updates: `via Atom <https://github.com/it-projects-llc/pos-addons/commits/11.0/.atom>`_, `by Email <https://blogtrottr.com/?subscribe=https://github.com/it-projects-llc/pos-addons/commits/11.0/.atom>`_
 
-Tested on Odoo 10.0 49ca43d75cb9a97642c820c2466d454f1ce604cb
+Tested on Odoo 11.0 06569efca35ef1eee119a1f9c3f73cf79fb70718
```
Comment (em230418) [inline pos_invoice_postponed/README.rst]: Нет такого коммита. https://github.com/odoo/odoo/commit/06569efca35ef1eee119a1f9c3f73cf79fb70718

**Review by em230418 — CHANGES_REQUESTED** [review]
Исправить мелочь

**Review by kaadevelop — COMMENTED** [review]
1 modules is updated:

 Updated files in 1 installable modules:
```
└─ pos_invoice_postponed/
   ├─ README.rst
   ├─ __manifest__.py
   ├─ models/
   |  └─ models.py
   ├─ static/
   |  ├─ description/
   |  |  └─ index.html
   |  └─ src/
   |     └─ js/
   |        └─ main.js
   └─ tests/
      └─ test_default.py
```

 Updated files in 0 non-installable modules:
are absent

You can use this review-bot too. Look: https://odoo-devops.readthedocs.io/en/latest/

No new features to test

File: pos_invoice_postponed/README.rst (hunk 1 above)
Comment (fedoranvar) [reply pos_invoice_postponed/README.rst]: исправил

**Review by em230418 — APPROVED** [review]

**Pull request merged**